fedora/sonarr.spec

```diff
@@ -0,0 +1,100 @@
+%define sonarr_url_service  http://services.nzbdrone.com/v1/update/master
+# default mock chroots do not provide python
+#%define sonarr_url_package  %( python -c 'import requests; print(requests.get("%{sonarr_url_service}").json()["updatePackage"]["url"].replace("windows","mono").replace("zip","tar.gz"))' )
+#%define sonarr_version      %( python -c 'import requests; print(requests.get("%{sonarr_url_service}").json()["updatePackage"]["version"])' )
+%define sonarr_url_package  %( curl -s "%{sonarr_url_service}" | tr -d '[[:space:]]' | sed -n 's#.*"url":"\\([^"]*\\)".*#\\1#p' | sed 's#windows#mono#g;s#zip$#tar.gz#' )
+%define sonarr_version      %( curl -s "%{sonarr_url_service}" | tr -d '[[:space:]]' | sed -n 's#.*"version":"\\([^"]*\\)".*#\\1#p' )
+
+
+Name:           sonarr
+Version:        %{sonarr_version}
+
+Release:        1%{?dist}
+Summary:        PVR for Usenet and BitTorrent users
+
+License:        GPLv3+
+URL:            https://sonarr.tv/
+Source0:        %{sonarr_url_package}
+Source1:        copyright
+Source2:        license
+Source3:        sonarr.systemd
+
+Requires:           sqlite >= 3.7
+Requires:           mediainfo >= 0.7.52
+Requires:           mono-complete >= 3.10
```

Comment:
For the most part it does, except:

* libmono-cil-dev has no fedora counterpart. The packages available are @ [Xamarin Mono Repo](http://download.mono-project.com/repo/centos/RPMS/).
* sonarr's wiki recommends mono 3.10, I think this is a good idea, and it's the only mono-3 branch available to fedora.